src/Connection.h

```diff
@@ -85,6 +102,7 @@ class Connection : public QObject {
 		/// Look up the local port of this Connection.
 		quint16 localPort() const;
 		bool bDisconnectedEmitted;
+		VoiceProtocolType vptVoiceProtocolType = VoiceProtocolType::UNDEFINED;
 
```

Comment:
This field is used for storing the protocol that is being used. So why not use VoiceProtocol? Because VoiceProtocol is a container for both official and custom protocols.

Consider the fact that _Murmur is the official one_, I think I don't have to deal with custom protocols in Murmur.

Reply:
Well I think it might still be nice to have the possibility in order to introduce experimental protocols and release them. However before we are sure we want to keep it, we won't allocate an actual ID in the protocol spec for it. Thus being able to use custom ones in murmur as well might be valuable :thinking:

Reply:
That makes some sense and is not hard either. Will do.